codex-ui/src/vue/components/input/Input.vue

```diff
@@ -41,14 +54,28 @@ onMounted(() => {
 });
 </script>
 
-<style lang="postcss" module>
+<style module>
 .input {
-  background-color: var(--base--bg-secondary);
+  display: flex;
   border: 0;
   outline: 0;
```

Comment:
div has no border and outline, so these lines has no sense

Reply:
There is a small problem: if we remove `border: 0;`, a border will appear around the parent container, so I suggest removing this line

![image](https://github.com/codex-team/notes.web/assets/94054053/e18e0d7d-79be-41a0-b4f9-ac1bccd60fa0)